packages/blockchain-link/src/ui/config.ts

```diff
@@ -430,4 +429,48 @@ export default [
             subscribe: '',
         },
     },
+    {
+        blockchain: {
+            name: 'Cardano Mainnet',
+            worker: 'js/cardano-worker.js',
+            server: ['wss://trezor-cardano-mainnet.blockfrost.io'],
```

Comment:
Looking at https://blockfrost.io/#pricing it looks like there're some limits on the free tier. How are we gonna ensure trezor (or suite to be specific) won't hit these limits? Does this `trezor-cardano-mainnet` do that? If so, how does blockfrost prevents others to abuse it?

Reply:
The Trezor Blockfrost backends are separate hosted instances, it is not tied to the free/public tier. It is rate limited to the specific end user by session and IP address, same as the Trezor Blockbook backends.

Reply:
@mmahut Thanks, that answers my concerns. .) Resolving.